src/components/DropArea.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from 'react';
+import 'react-dropzone';
+import Dropzone from 'react-dropzone';
+
+class DropArea extends Component {
+  static displayName = "DropArea";
+  state = {
+    count : 0
+  }
+  onDrop (acceptedFile, rejectedFiles) {
+    this.setState({
+    files: acceptedFile
+    });
+  }
+  //    onOpenClick () {
+  //      this.dropzone.open();
+  //    }
+  render() {
+    return (
+      <div>
+          <Dropzone ref={(node) => { this.dropzone = node; }} onDrop={this.onDrop}>
+              <div>Try dropping some files here, or click to select files to upload.</div>
+          <button type="button" onClick={this.onOpenClick}>
+              Open Dropzone
+          </button>
+          {this.state && this.state.files && this.state.files.length > 0 ? <div>
```

Comment:
I think we should avoid documenting and commenting if possible. I can re-factor the format to make it more clear though, I agree that the formatting is a bit funny. This is mostly taken from the example app provided by the drop zone plugin.